cwl/src/main/scala/cwl/OutputParameter.scala

```diff
@@ -0,0 +1,124 @@
+package cwl
+
+import cats.syntax.traverse._
+import cats.syntax.validated._
+import common.validation.ErrorOr._
+import shapeless.{:+:, CNil, Poly1}
+import wom.types.WomFileType
+import wom.values.{WomFile, WomString, WomValue}
+
+trait OutputParameter {
+  def id: String
+  def label: Option[String]
+  def secondaryFiles: Option[SecondaryFiles]
+  def format: Option[StringOrExpression]
+  def streamable: Option[Boolean]
+  def doc: Option[String :+: Array[String] :+: CNil]
+  def outputBinding: Option[CommandOutputBinding]
+  def `type`: Option[MyriadOutputType]
+}
+
+object OutputParameter {
```

Comment:
Everything (except the unapplies) here is just copied over from `CommandLineTool`